### ISSUE TITLE
Failed to build for targetSdkVersion and compileSdkVersion 31

### ISSUE THREAD

**Issue by lohanbodevan** [Baseflow/flutter-geocoding#87]
## 🐛 Bug Report

When trying to build Flutter app targeting and compiling for Android SDK version 31, build fails with the following error

```bash
* What went wrong:
Execution failed for task ':geocoding:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/Users/xxxx/.gradle/caches/transforms-3/d647e4cf153df5c584ca88f6d0c0ffec/transformed/core-1.7.0/res/values/values.xml:105:5-114:25: AAPT: error: resource android:attr/lStar not found.
```

### Configuration

**Version:** 2.0.2
**Flutter**: 2.10.1 

**Platform:** 
- [ ] :iphone: iOS
- [x] :robot: Android

**Comment by josechagas** [comment]
I am having the same problem

**Comment by alanbosco003** [comment]
Am also having the same issue

**Comment by ItsYasser** [comment]
same issue this should be fixed asap

**Comment by mvanbeusekom** [comment]
This morning I released version 2.0.3 of the geocoding plugin which should resolve this issue.
